src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -53,6 +53,13 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
   );
   const purchaseHistory = useSelector(userPurchaseHistorySelector);
 
+  const redirectUrl = venue?.config?.redirectUrl ?? "";
```

Comment:
There is no need to set a default value with ?? here. It’s semantically more useful/correct that if it isn’t set, that the variable remains undefined. By setting it to an empty string we’re just kind of shadowing/hiding the reality of it.

in this specific case it doesn’t make a huge difference to the logic/flow of things as it’s checked right below, but this is n anti-pattern that can lead to much bigger/more annoying/harder to track down bugs, so calling it out here so people can learn and make sure not to repeat it on a winder scale.